tests/functional/c/consider/consider_using_f_string.py

```diff
@@ -40,6 +40,11 @@ def print_good():
     print("%s" % "\n")
     print("%s" % "\n".join(i for i in "string"))
 
+    def wrap_print(value):
```

Comment:
I think we can move this to the bottom of this file. Perhaps with a comment saying:
`Non valid code that shouldn't crash the checker`.
That would decrease the size of this diff as all line numbers don't change.